### PR TITLE
Fix #749: Display heatmap y-axis label correctly

### LINKED ISSUE #749: heatmap y label not correctly displayed
**问题**

The y label is not correctly displayed in heatmap. Expected behavior is showing the label (category), however it shows the index (integer starting from 0).

**运行环境（系统环境及 pyecharts 版本）**

Windows 10, pyecharts version 0.5.10
Probably system independent.

**代码及截图**

To reproduce:

```python
arr = np.empty([10, 20])
for i in range(10):
    for j in range(20):
        arr[i, j] = i**2 - j

heatmap = pyecharts.HeatMap(height=400, width=500)
heatmap.add(
    'heatmapasdkfjldjflsdjf',
    list('abcdefghij'),
    list('abcdefghijabcdefghij'),
    [[i, j, arr[i, j]] for i in range(10) for j in range(20)],
    is_visualmap=True,
)

heatmap
```

**Temporary fix**

In pyecharts\echarts\axis.py, line 32, change:

```python
        if not isinstance(formatter, types.FunctionType):
            formatter = "{value} " + formatter
        super(YAxisLabel, self).__init__(formatter=formatter, **kwargs)
```

to

```python
        super(YAxisLabel, self).__init__(**kwargs)
```

A related question is why is X axis label defined slightly differently (just above the y part)? Is the temporary fix an ideal solution?

### PR DESCRIPTION
**Pull request by thombashi**
I believe the PR will fix #749 and do no harm to other formats.

The following example shows before and after the modification (extracted from https://github.com/pyecharts/pyecharts-users-cases/blob/master/notebook-users-cases/notebook-user-cases.ipynb):

## Before the fix
before

## After the fix
after

### CODE REVIEW

**Comment by chfw** [comment]
The fix looks good. Could you please contribute a unit test so that your change will be permanent?

**Comment by thombashi** [comment]
@chfw 
Thank you for your feedback.
I had added some test cases.
Please check it out.

File: test/test_heatmap.py [hunk 1]
```diff
@@ -29,6 +29,45 @@ def test_heatmap_default():
...
+        Y_WEEK,
+        data,
+        is_visualmap=True,
+        is_label_show=False,
+        visual_text_color="#000",
+        visual_orient='horizontal',
+        yaxis_formatter="",
+    )
+    html_content = heatmap._repr_html_()
+    assert r'"formatter": "{value} "' in html_content
```
Comment (chfw) [inline test/test_heatmap.py]: looks like  with or without yaxis_formatter, the assertion is the same.

File: pyecharts/echarts/axis.py [hunk 2]
```diff
@@ -29,7 +29,10 @@ def __init__(self, **kwargs):
 
 class YAxisLabel(AxisLabel):
     def __init__(self, formatter=None, **kwargs):
-        if not isinstance(formatter, types.FunctionType):
+        if (
```
Comment (chfw) [inline pyecharts/echarts/axis.py]: YAxisLabel could have one small unit test.

File: test/test_heatmap.py (hunk 1 above)
Comment (thombashi) [reply test/test_heatmap.py]: `"formatter": "{value} "` will not included to `"axisLabel"` in the HTML if `yaxis_formatter` not specified at the `heatmap.add` method call, in this case, y-axis labels are properly displayed.
On the other hand, `"formatter": "{value} "` will included in the HTML if `yaxis_formatter` specified other than `None`, even if the value is empty string(`""`) . This case, y-axis displays y-values (0-6).
Does it makes sense?

File: pyecharts/echarts/axis.py (hunk 2 above)
Comment (thombashi) [reply pyecharts/echarts/axis.py]: I had added unit tests for `YAxisLabel` to the `test/test_axis.py` file.

File: test/test_heatmap.py (hunk 1 above)
Comment (chfw) [reply test/test_heatmap.py]: Alright, so to get week day names display, yaxis_formatter should be given "".

**Pull request merged**